posthog-core/src/surveys-types.ts

```diff
@@ -1,3 +1,7 @@
+// Forces this file to be treated as a module
+// See https://github.com/PostHog/posthog-js-lite/issues/448
+export {}
```

Comment:
Other solutions are to change all `export` to `declare`, or wrap the whole class in a `declare global`.
The quick `export {}` seems to be simpler.